ledger-service/http-json/src/main/scala/com/digitalasset/http/HttpService.scala

```diff
@@ -371,7 +359,7 @@ object HttpService {
 
   private def uploadDarAndReloadPackages(
       f: LedgerClientJwt.UploadDarFile,
-      g: LoggingContextOf[InstanceUUID] => Future[PackageService.Error \/ Unit],
+      g: Jwt => LoggingContextOf[InstanceUUID] => Future[PackageService.Error \/ Unit],
   )(implicit ec: ExecutionContext): LedgerClientJwt.UploadDarFile =
-    (x, y) => implicit lc => f(x, y)(lc).flatMap(_ => g(lc).flatMap(toFuture(_): Future[Unit]))
+    (x, y) => implicit lc => f(x, y)(lc).flatMap(_ => g(x)(lc).flatMap(toFuture(_): Future[Unit]))
```

Comment:
another place IMO where we can probably use slightly more verbose variable and function names :)